update managedcluster clusterrole and managedclusterset clusterrole

ldpliu opened the pull request:
https://github.com/open-cluster-management/backlog/issues/10682
https://github.com/open-cluster-management/backlog/issues/10681

ldpliu committed:
update managedcluster clusterrole and managedclusterset clusterrole

dockerymick commented on authentication/rbac.adoc:
```diff
@@ -30,12 +30,18 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
+| *open-cluster-management:view:managed-cluster-x*
 | A user with cluster-wide binding to the `managed-cluster-x` role has view access to `managedcluster “X”` resource.
 
+| *open-cluster-management:managedclusterset:admin:managed-clusterset-x*
```
@ldpliu I made a couple of changes. Let me know your thoughts.

dockerymick reviewed (approved):
Thanks for your updates. I made a couple of changes and left a comment in case you had an opinion about my changes. Otherwise, this looks good to me! If you agree with the updates, feel free to merge

qiujian16 commented on authentication/rbac.adoc:
```diff
@@ -30,12 +30,18 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
+| A user with cluster binding to the `managed-clusterset-x` role has administrator access to `managedclusterset “X”` resource. This role also gives the user administrator access to all resources that are in the `managedclusterset “X”` resource. 
+
+| *open-cluster-management:managedclusterset:view:managed-clusterset-x*
+| A user with cluster-wide binding to the `managed-clusterset-x` role has view access to `managedclusterset “X”` resource. This role also gives the user view access to all resources that are in the `managedclusterset “X”` resources.
```
it is hard to understand what is the  resources in managedclusterset.  I would explicitly  set user can view managedcluster/clusterclaim.hive.openshit.io/clusterdeployment/clusterpool in with the cluster set label of "open-clusterr-management.io/clusterset=x"

dockerymick replied:
this was addressed by @ldpliu, checking this conversation as resolved

dockerymick committed:
Update rbac.adoc

Made a few more commits

dockerymick commented:
@ldpliu I made a few more commits. I replaced "-x" with `-<managed_clusterset_name>` or `-<managed_cluster_name>` I have this update in my active [PR](https://github.com/open-cluster-management/rhacm-docs/pull/2011). James suggested this change so that it is more clear on what info is needed.

dockerymick commented on authentication/rbac.adoc:
```diff
@@ -30,11 +30,17 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
+| A user with cluster-wide binding to the `managed-cluster-<managed_cluster_name>` role has view access to `managedcluster-<managed_cluster_name>` resource.
+
+| *open-cluster-management:managedclusterset:admin:managed-clusterset-<managed_clusterset_name>*
+| A user with cluster binding to the `managed-clusterset-<managed_clusterset_name>` role has administrator access to `managedclusterset-<managed_clusterset_name>` resource. The user also has administrator access to `managedcluster.cluster.open-cluster-management.io/clusterclaim.hive.openshit.io/clusterdeployment.hive.openshit.io/clusterpool.hive.openshit.io` resources, which has the managed clust ...
```
@ldpliu is this a directory path or were you trying to list the different resources? If you are listing the different resources, i recommend using commas (,) instead of the backslashes (/)

dockerymick replied:
Also, are the role names and resource names the same?

dockerymick replied:
The first comment was partially addressed by @qiujian16 in his comment here: https://github.com/open-cluster-management/rhacm-docs/pull/2026#discussion_r622901663. I made the decision to use commas instead of backslashes. 

The second comment was addressed during the meeting on the 29th at 9 PM by @ldpliu.

qiujian16 commented on authentication/rbac.adoc:
```diff
@@ -30,11 +30,17 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
+| A user with cluster-wide binding to the `managed-cluster-<managed_cluster_name>` role has view access to `managedcluster-<managed_cluster_name>` resource.
+
+| *open-cluster-management:managedclusterset:admin:managed-clusterset-<managed_clusterset_name>*
+| A user with cluster binding to the `managed-clusterset-<managed_clusterset_name>` role has administrator access to `managedclusterset-<managed_clusterset_name>` resource. The user also has administrator access to `managedcluster.cluster.open-cluster-management.io/clusterclaim.hive.openshit.io/clusterdeployment.hive.openshit.io/clusterpool.hive.openshit.io` resources, which has the managed clust ...
```
A user with cluster binding to the `managed-clusterset-<managed_clusterset_name>` role has administrator access to the  ManagedCluster resource with the name of <managed_clusterset_name>

dockerymick replied:
Addressed this update

dockerymick committed:
Update rbac.adoc

Made some updates for the resource names and used commas instead of backslashes (/) to separate the resources that the role has access to.

dockerymick commented:
@ldpliu @qiujian16 may you all do another scan of this PR before we decide to merge? I made a few more commits

ldpliu commented:
close this pr and trace in https://github.com/open-cluster-management/rhacm-docs/pull/2011/files#diff-9d6a0a80be90a5f434583a1c1fc7f44c3ff504025db03bddcc62d1f809d8a44f
